aws/provider.go

```diff
@@ -483,6 +483,7 @@ func Provider() *schema.Provider {
 			"aws_cloudformation_stack_set":                            resourceAwsCloudFormationStackSet(),
 			"aws_cloudformation_stack_set_instance":                   resourceAwsCloudFormationStackSetInstance(),
 			"aws_cloudfront_distribution":                             resourceAwsCloudFrontDistribution(),
+			"aws_cloudfront_key_group":                                resourceAwsCloudFrontKeyGroup(),
```

Comment:
We can changelog this by adding a `.changelog/17041.txt` file with contents such as:

``````markdown
```release-note:new-resource
aws_cloudfront_key_group
```
``````